src/index.html

```diff
@@ -1,17 +1,169 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <div class="group">
+        <fieldset>
+          <legend>Personal information</legend>
+          <label class="personal">
+            Surname:
+            <input
+            type="text"
+            autocomplete="off"
+            name="Surname"
+            required
+            >
+          </label>
+          <label class="personal">
+            Name:
+            <input
+            type="text"
+            autocomplete="off"
+            name="Name"
+            required
+            >
+          </label>
+          <label class="personal">
+            How old are You?
+            <input
+            type="number"
+            value="12"
+            autocomplete="off"
+            min="1"
+            max="100"
+            name="Old"
+            required
+            >
+          </label>
```

Comment:
1) apply code styling recommendations all over the page
2) apply autocomplete="off" only to the text inputs
3) write input `name` lowercased 
4) give more meaningful names `old` -> `age` 
5) add common class to all your fields like `field`
6) apply those recommendations all over the page
```suggestion
          <label class="field">
            How old are You?
            <input
              type="number"
              value="12"
              min="1"
              max="100"
              name="age"
              required
            >
          </label>
```